integration/servicenow-integration/src/main/java/se/inera/intyg/webcert/integration/servicenow/stub/config/ServicenowStubConfig.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2024 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package se.inera.intyg.webcert.integration.servicenow.stub.config;
+
+import static se.inera.intyg.webcert.integration.api.subscription.ServiceNowIntegrationConstants.SERVICENOW_INTEGRATION_STUB_PROFILE;
+
+import com.fasterxml.jackson.jaxrs.json.JacksonJsonProvider;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.cxf.Bus;
+import org.apache.cxf.bus.spring.SpringBus;
+import org.apache.cxf.endpoint.Server;
+import org.apache.cxf.jaxrs.JAXRSServerFactoryBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.context.support.PropertySourcesPlaceholderConfigurer;
+import org.springframework.core.io.ClassPathResource;
+import se.inera.intyg.webcert.integration.servicenow.stub.api.ServiceNowStubRestApi;
+import se.inera.intyg.webcert.integration.servicenow.stub.api.ServiceNowStubSettingsApi;
+
+@Configuration
+@Profile(SERVICENOW_INTEGRATION_STUB_PROFILE)
+@ComponentScan(basePackages = "se.inera.intyg.webcert.integration.servicenow.stub")
+public class ServicenowStubConfig {
```

Comment:
Should be ServiceNow.... not Servicenow...

Reply:
done